observability/readme.md

```diff
@@ -43,6 +43,8 @@ As much as possible, configuration and management of observability assets such a
 
 3. Automating Log Analytics Queries - There are several use cases where automation of log analytics queries may be needed. Example, Automatic Report Generation, Running custom queries programmatically for analysis, debugging etc. For these use cases to work, log queries should be source controlled and automation can be built using [log analytics REST](https://dev.loganalytics.io/documentation/Using-the-API) or [azure cli](https://docs.microsoft.com/en-us/cli/azure/ext/log-analytics/monitor/log-analytics?view=azure-cli-latest).
 
+4. Microservices is a very popular software architecture, where the application is arranged as a collection of loosely coupled services. Some of those services can be written in different languages by different teams. Learn more [here](microservices.md).
```

Comment:
This section might not be required here, as it is mostly talking about the "infrastructure" related to observability that can be written maintained as code.

Reply:
Where do you suggest to place this?